sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -2821,14 +2821,18 @@ SIREPO.app.directive('sbatchLoginModal', function() {
               '</div>',
             '</div>',
         ].join(''),
-        controller: function(requestSender, $scope) {
-            $scope.password = "";
-            $scope.username = "";
+        controller: function(requestSender, $scope, $element) {
+            $scope.form = angular.element($($element).find('form').eq(0));
+            $scope.password = '';
+            $scope.username = '';
```

Comment:
`$scope.otp = "";`

Also `type="password"` on the input field since it is a password